cms/djangoapps/cms_user_tasks/tasks.py

```diff
@@ -22,6 +22,14 @@ def send_task_complete_email(self, task_name, task_state_text, dest_addr, detail
     """
     Sending an email to the users when an async task completes.
     """
+    disable_emails = configuration_helpers.get_value(settings.DISABLE_CMS_TASK_EMAILS, default='true') == 'true'
```

Comment:
@rehan99000 for boolean settings we use `FEATURES` like this
```  
enable_mktg_site = configuration_helpers.get_value(
        'ENABLE_MKTG_SITE',
        settings.FEATURES.get('ENABLE_MKTG_SITE', False)
    )
```

Reply:
@ziafazal `default` is a kwarg so I put it like that. Since it is not inside `FEATURES`, `settings.get(settings.DISABLE_CMS_TASK_EMAILS, True)` would not make sense. do let me know if I misunderstood u.

Reply:
@rehan99000 I pasted example of how  value of `enable_mktg_site` variable is set. We need same approach here.

Reply:
@ziafazal is this okay?
```
disable_emails = configuration_helpers.get_value(
        settings.DISABLE_CMS_TASK_EMAILS,
        'true'
    ) == 'true'
```